ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -78,29 +79,29 @@ ansible_collections/arista/avd/examples/campus-fabric/switch-basic-configuration
 Now that we understand the physical L2LS topology, we must create the Ansible inventory that represents this topology. The following is a textual and graphical representation of the Ansible inventory group variables and naming scheme used in this example:
 
 ``` text
-- DC1
-  - DC1_FABRIC
-    - DC1_SPINES
-    - DC1_LEAFS
-  - DC1_NETWORK_SERVICES
-    - DC1_SPINES
-    - DC1_LEAFS
-  - DC1_NETWORK_PORTS
-    - DC1_SPINES
-    - DC1_LEAFS
+- ACME
+  - S1
+    - S1_SPINES
+    - S1_LEAFS
+  - NETWORK_SERVICES
+    - S1_SPINES
+    - S1_LEAFS
+  - CONNECTED_ENDPOINTS
+    - S1_SPINES
+    - S1_LEAFS
 ```
 
-DC1 represents the highest level within the hierarchy. Ansible variables defined at this level will be applied to all nodes in the fabric. Ansible groups have parent-and-child relationships. For example, both DC1_SPINES and DC1_LEAFS are children of DC1_FABRIC. Groups of Groups are possible and allow variables to be shared at any level within the hierarchy. For example, DC1_NETWORK_SERVICES is a group with two other groups defined as children: DC1_SPINES and DC1_LEAFS. The same applies to the group named DC1_NETWORK_PORTS. You will see these groups listed at the bottom of the inventory file.
+ACME represents the highest level within the hierarchy. Ansible variables defined at this level will be applied to all nodes in the fabric. Ansible groups have parent-and-child relationships. For example, both S1_SPINES and S1_LEAFS are children of S1. Groups of Groups are possible and allow variables to be shared at any level within the hierarchy. For example, NETWORK_SERVICES is a group with two other groups defined as children: S1_SPINES and S1_LEAFS. The same applies to the group named CONNECTED_ENDPOINTS. You will see these groups listed at the bottom of the inventory file.
 
 This naming convention makes it possible to extend anything quickly and can be changed based on your preferences. The names of all groups and hosts must be unique.
 
 ![Figure: 2](images/ansible_groups.svg)
 
 ### inventory.yml
 
-The below inventory file represents two spines and ten leafs. The nodes are defined under the groups DC1_SPINES and DC1_LEAFS, respectively. We apply group variables (group_vars) to these groups to define their functionality and configurations.
+The below inventory file represents two spines and ten leafs. The nodes are defined under the groups S1_SPINES and S1_LEAFS, respectively. We apply group variables (group_vars) to these groups to define their functionality and configurations.
```

Comment:
I think there's only 8 leafs, not 10 right?

Reply:
Correct, fixing now